otter/integration/lib/autoscale.py

```diff
@@ -194,10 +204,10 @@ def wait_for_N_servers(
             Otherwise, an exception is raised.
         """
 
-        def check(state):
-            if state[0] != 200:
+        def check((code, response)):
```

Comment:
Don't get me wrong, the destructuring is way better than `state[n]`; it should probably just not happen in the argspec :)